### PR TITLE
tracker: toss.im

### PR DESCRIPTION
**Pull request by seia-soto**
refs https://github.com/ghostery/trackerdb/pull/189

### CODE REVIEW

File: youslist.txt [hunk 1]
```diff
@@ -2186,6 +2186,7 @@
 ||torrentpi*.com/data/file/ad/
 ||torrentpi*.com/data/file/ads/
 ||torrentsee*.com/js/jquery.ksad.js^
+||toss.im/pages/ekdil/tragoudo^
```
Comment (yous) [inline youslist.txt:2189]: If this is used only in `toss.im`, can you add `$domain=toss.im` to this line?

Comment (seia-soto) [reply youslist.txt:2189]: I didn't check other sites but it should be ok.

**Comment by seia-soto** [comment]
@yous do you want to merge all PRs into one? I can make things simple.

**Comment by yous** [comment]
It's okay. I'll update Rules.1blockpkg in each PR, so feel free to open multiple PRs.

**Comment by yous** [comment]
Thanks!

**Pull request merged**